<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QToolButton" name="saveToolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>save_ico.png</normaloff>save_ico.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="exitToolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>exit_ico.png</normaloff>exit_ico.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="originalPartWidget" native="true">
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
       <item row="0" column="1">
        <widget class="QListWidget" name="originalListWidget">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>458</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>925</width>
           <height>10000000</height>
          </size>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="translatedPartStackedWidget">
      <property name="sizeIncrement">
       <size>
        <width>2</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="listPage">
       <layout class="QGridLayout" name="gridLayout_2" rowminimumheight="0">
        <item row="0" column="0">
         <widget class="QListWidget" name="translatedListWidget">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>458</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>925</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="editorPage">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveBlockPushButton">
            <property name="text">
             <string>Сохранить блок</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="translateApiPushButton">
            <property name="text">
             <string>Перевести</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="originalTextEdit">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QTextEdit" name="translatedTextEdit">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="workWithBlockPushButton">
        <property name="text">
         <string>Перевести блок</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Экспорт</string>
     </property>
     <addaction name="separator"/>
     <addaction name="exportTxtTrigger"/>
     <addaction name="separator"/>
     <addaction name="exportPdfTrigger"/>
    </widget>
    <addaction name="createTrigger"/>
    <addaction name="separator"/>
    <addaction name="openTrigger"/>
    <addaction name="separator"/>
    <addaction name="saveTrigger"/>
    <addaction name="separator"/>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="exitTrigger"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="createTrigger">
   <property name="text">
    <string>Новый Проект</string>
   </property>
  </action>
  <action name="openTrigger">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveTrigger">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="exitTrigger">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="exportTxtTrigger">
   <property name="text">
    <string>TXT</string>
   </property>
  </action>
  <action name="actionPDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="exportPdfTrigger">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
